package.json

```diff
@@ -46,7 +46,10 @@
     "nav-frontend-alertstriper": "1.0.27",
     "nav-frontend-alertstriper-style": "0.2.48",
     "nav-frontend-brodsmuler-design": "2.0.0",
-    "nav-frontend-core": "4.0.5",
+    "nav-frontend-chevron-style": "^0.3.4",
```

Comment:
Det vil være bedre hvis dere bruker Team Sykefraværs versjon av denne: `import { Utvidbar } from 'digisyfo-npm'` – dette for å minimere antall avhengigheter. Da kan dere ta bort en hel del av de avhengighetene dere har introdusert.